### ISSUE TITLE
Contact Center Review for Debt Letters MVP  + Iteration2 

### ISSUE THREAD

**Issue by leahkeeler** [department-of-veterans-affairs/va.gov-team#12948]
Request the Contact Center Review 

## What this form is for
Use this template to request a Contact Center review of your product.

Please review the Self service Product Guide Template [link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/self-service-product-guide-template.md) and the Contact Center Review How to [link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/request-contact-center-review.md)
 

## Description
Debt Letters MVP allows the veterans to see debt they have with Debt Management Center (DMC) and download associated debt letters. 

### Launch Date
Phased release starting 9/14

[Product Guide](https://app.zenhub.com/files/133843125/c6c9f39c-2d23-4ffe-8d30-bd68ff659c8b/download) 

[Product Video Transcript ](https://app.zenhub.com/files/133843125/b9ed550a-1da1-4ba2-bd2c-6fc00bfa2395/download)

[Video](https://drive.google.com/file/d/1PqjJrMFGhe5N9t086IZgspylso7gd62f/view?usp=sharing)

## Acceptance Criteria
- [ ] VSP Analytics & Insights QA has been completed 
- [ ] VSP Analytics & Insights sends artifacts to appropriate contact centers
- [ ] VFS team completes brief VSP Collaboration Cycle Feedback survey [link](https://adhoc.optimalworkshop.com/questions/20260uu8-0-0/questions/before)

**Comment by leahkeeler** [comment]
Video was too large to upload to Github - it is a google drive link, please let me know if you are having any issues opening, downloading. Thanks!

**Comment by Kimberley2019** [comment]
Good Afternoon @leahkeeler I will review and send along to VEO for their review and feedback.  Can you please confirm that this is the most up to date product guide and video and that there are no additional changes to be made in the foreseeable future?

**Comment by leahkeeler** [comment]
Kimberly, Thanks for reaching out -- Can you please hold on this request, we just found a major blocker that may change the product guide. Thank-you so so much - Leah

**Comment by Kimberley2019** [comment]
Absolutely Leah.  Just keep us posted.

**Comment by leahkeeler** [comment]
Kimberly, This is ready to go- we have a launch date :) Thanks, Leah

**Comment by clantosswett** [comment]
@leahkeeler These items have been reviewed and sent over to the contact center. They might have some follow up questions but otherwise, we have what we need. However, would it be possible to provide more detail for the launch plan so we can update the contact center accordingly. Thanks!

**Comment by leahkeeler** [comment]
HI @clantosswett -I am sorry to say this release was pushed to 1/1/2021. We are continuing to add features to this product. Once we get closer to release date we will see if we keep the videos and such by iteration or if it makes sense to combine. I am going to move this to our backlog for now. 

Thanks for all your help :) Leah

**Comment by Kimberley2019** [comment]
@leahkeeler Thank you for the update.  I have passed this information along to the Contact Center today.
On hold.

**Comment by joanneesteban** [comment]
Thanks, everyone! Moving to backlog on VSP.

**Comment by leahkeeler** [comment]
Kimberly, Debt Letters MVP will be super-ceded by Debt Letters Iteration 2, scheduled for release first 2 weeks of January (date TBD based on PTO schedule). 

We are in process of updating the product guide and video and will ping you once ready for review. As you remember MVP was not released for political reasons.  Thanks! Leah

**Comment by Kimberley2019** [comment]
Thank you for the heads up Leah.  We will be awaiting your ping.

**Comment by 3obryandogs** [comment]
@leahkeeler Attached are files for the I2 video script and user guide.

[Leah's video script - updated.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5645949/Leah.s.video.script.-.updated.docx)

[Leah's product guide - updated.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5645946/Leah.s.product.guide.-.updated.docx)

**Comment by leahkeeler** [comment]
HI @3obryandogs - the product guide still has screen shots from Iteration 1 - it needs ot be updated to iteration 2

**Comment by 3obryandogs** [comment]
@leahkeeler - I used screen shots from https://staging.va.gov/manage-va-debt/your-debt/. I left in two that you'd made earlier because they had arrows and notes. Let me know what you want changed. Thanks!

**Comment by Kimberley2019** [comment]
@leahkeeler I've been seeing some movement on this again.  Please be sure to ping me with the required information in order for us to review and pass it along to the contact center for their review.

**Comment by leahkeeler** [comment]
HI @Kimberley2019 I am finishing this up today and will have it to you

**Comment by Kimberley2019** [comment]
No worries at all.  Not rushing you, just don't want to miss it on my end.

**Comment by leahkeeler** [comment]
Hi Kimberly, All the content has been updated for Iteration 2, scheduled to be released on va.gov on January 4, 2021. Please note that the video was too large to upload to github and is linked to google docs instead -- LMK if you have any issues downloading.

**Comment by leahkeeler** [comment]
Hi @Kimberley2019 This is launched - are you able to review?

**Comment by starlow00** [comment]
@Kimberley2019 Attached are the Product Guide and Video Transcript. The Product Guide requires some edits, while the Video Transcript doesn't have edits that require a re-record, unless the year referenced vis a vis COVID-19 needs to change to something in 2021 (noted on attached Word doc). Questions, please let me know!

[12948 CC Debt Letter MVP Video Transcript_ST_011921.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5838595/12948.CC.Debt.Letter.MVP.Video.Transcript_ST_011921.docx)
[12948 CC Debt Letters MVP Product Guide_ST_011921.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5838596/12948.CC.Debt.Letters.MVP.Product.Guide_ST_011921.docx)

**Comment by Kimberley2019** [comment]
@leahkeeler First let me offer my sincerest apologies for the delay on this ticket.  I was OOO for some time.

Please review the edits that our technical writer has suggested and advise if they can be incorporated.  Once you have reviewed, we will then send it to the VEO contact center for review and any feedback.

If you indicate that you would prefer not to incorporate the suggestions, we will send to VEO without those updates and wait for their feedback.  Of course I provide a deadline to hear back from them.

**Comment by leahkeeler** [comment]
Thank-you for the edits - I have been working on this and will have it to you by early next week - Thanks again! Leah

**Comment by leahkeeler** [comment]
Hi @Kimberley2019 @jwoodman5 We just received news that debt collection may be suspended per EO signed on Friday - we have removed the debt restarting alert, I am going to hold the product guide until we have concrete direction on if and when debt suspension is restarting and what the new alert will contain - FYI

**Comment by jwoodman5** [comment]
Thank you for the head's up, @leahkeeler!

**Comment by Kimberley2019** [comment]
@leahkeeler I have moved this ticket to Icebox.  Please let us know what else needs to be done or ping us once this is ready for movement again. Thanks!

**Comment by leahkeeler** [comment]
Thanks Kimberly -hoping ot have some resolution or path forward this week
